README.adoc

```diff
@@ -488,10 +488,16 @@ include::finish/inventory/src/main/java/io/openliberty/guides/inventory/Inventor
 
 In order to access the Tracer, the [hotspot=inject file=0]`@Inject` annotation from the Contexts and Dependency Injections API injects the Tracer into a bean. 
 
-Before the [hotspot=manager hotspot=getSystem file=0]`InventoryManager` calls the `system` service, the `spanBuilder()` and `startSpan()` Tracer APIs create and start a span called [hotspot=getPropertiesSpan file=0]`GettingProperties`.
+Before the [hotspot=manager hotspot=getSystem file=0]`InventoryManager` calls the `system` service, it creates and starts a span called [hotspot=getPropertiesSpan file=0]`GettingProperties` by using the `spanBuilder()` and `startSpan()` Tracer APIs.
+
+When starting a span, you also need to end it by calling `end()` on the span. If you don't end a span, it won't be recorded at all and won't show up in Jaeger. This code ensures that `end()` is always called by including it in a `finally` block.
 
 The [hotspot=try file=0]`try` block is called a `try-with-resources` statement that the [hotspot=scope file=0]`scope` object is closed at the end of the statement. It’s good practice to define custom spans inside such statements. Otherwise, any exceptions that are thrown before the span closes will leak the active span.
```

Comment:
I think this paragraph should be removed. The same information is given in the following two paragraphs.